src/handlers/comment-created-callback.ts

```diff
@@ -0,0 +1,45 @@
+import { Context, SupportedEvents } from "../types";
+import { addCommentToIssue } from "./add-comment";
+import { askQuestion } from "./ask-llm";
+import { CallbackResult } from "../types/proxy";
+import { bubbleUpErrorComment } from "../helpers/errors";
+
+export async function issueCommentCreatedCallback(
+  context: Context<"issue_comment.created", SupportedEvents["issue_comment.created"]>
+): Promise<CallbackResult> {
+  const {
+    logger,
+    env: { UBIQUITY_OS_APP_NAME },
+  } = context;
+  const question = context.payload.comment.body.trim();
+  const slugRegex = new RegExp(`@${UBIQUITY_OS_APP_NAME}`, "gi");
```

Comment:
just a quick thought: shouldn't we only match the begging of the comment for tag because somebody might tag the bot in the middle of the comment accidentally without wanting to ask a question, but on the other hand some people might ask a question like `how do i assign myself @bot`

Reply:
Good point. I've seen intended usage of it with the invocation within the body, so I'm unsure if it's a design choice or if we should enforce typical slash command structure

Reply:
@0x4007 rfc